docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/fix-code-issues.adoc

```diff
@@ -4,7 +4,7 @@
 
 == Fix Code Security Issues
 
-On *Projects*, you can remediate scan results across all code categories by adding issues to the fix cart to create a Pull Request (PR) with a suggested fix. For every issue found on Prisma Cloud console, you can view information like origin of the issue in a file or repository, policy violation, and suggestions to remediate the issue.
+On *Projects*, many scan results across code categories support automated remediation. You can add issues to the fix cart to create a Pull Request (PR) with the suggested fix. For all other issues, you can also view information like origin of the issue in a file or repository, policy violation, and non-automated suggestions to remediate the issue.
```

Comment:
We should list the scan results and the categories which support automated remediation.